src/main/java/net/infogram/api/ConnectionManager.java

```diff
@@ -31,6 +32,8 @@ public static HttpURLConnection sendRequest(final String baseUrl, final String r
         }
     }
 
+    private enum HttpMethod { GET, POST, PUT, DELETE }
```

Comment:
Please move any inner class or enum declarations to the top of the container class